clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/controllers/servergroup/EcsServerGroupController.java

```diff
@@ -0,0 +1,80 @@
+package com.netflix.spinnaker.clouddriver.ecs.controllers.servergroup;
```

Comment:
Let's also mind the whitespace and try to keep non-test classes below 120 char in line length.

Reply:
Removed a double blank line, and added line breaks to stay below 120 chars in line length